pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -74,6 +74,7 @@
   hoc_live: 'Hour of Code Live'
   hoc_live_sign_up_title: 'Sign up for Hour of Code Live'
   hoc_live_sign_up_message: 'Learn in real time alongside special guests! Students of all ages and abilities, including those without computers, are welcome. Join our mailing list to receive recordings of the episodes and resources to continue learning at home.'
+  hoc_live_episodes_link: "See past episodes [here](%{episodes_url})."
```

Comment:
is there a specific reason we're adding a new string rather than adding this extra content to the existing string?

Reply:
No specific reason. Would you recommend the latter?

Reply:
Just pushed a change with the latter